railties/lib/rails/application/configuration.rb

```diff
@@ -87,6 +87,18 @@ def initialize(*)
       # {configuration guide}[https://guides.rubyonrails.org/configuring.html#versioned-default-values]
       # for the default values associated with a particular version.
       def load_defaults(target_version)
+        # To introduce a change in behavior, follow these steps:
+        # 1. Add an accessor on the target object (e.g. the ActiveJob class for global Active Job config).
+        # 2. Set a default value there preserving existing behavior for existing applications.
+        # 3. In the section below corresponding to the next release of Rails, set the new value.
+        # 4. Add a commented out section in the new_framework_defaults file, setting the new value.
+        # 5. Update the guide on configuration.
```

Comment:
```suggestion
        # 4. Add a commented out section in the new_framework_defaults template, setting the new value.
        # 5. Update the guide in `configuring.md`
```

Reply:
Updated. I also realized I omitted the step of actually implementing the change, and its corresponding removal step, so I added that too.

Commits squashed too.